examples/spark/test.bats

```diff
@@ -63,7 +57,8 @@ SPARK_LOCAL_DIRS=/tmp/spark
 SPARK_LOG_DIR=$spark_log
 SPARK_WORKER_DIR=/tmp/spark
 SPARK_LOCAL_IP=127.0.0.1
-SPARK_MASTER_HOST=${master_ip}
+SPARK_MASTER_HOST=${master_host}
+JAVA_HOME=/usr/lib/jvm/default-java/
```

Comment:
Does it work to unset `JAVA_HOME`? I thought Java was smart about that if the variable wasn't already set. I.e., is the problem bogus `JAVA_HOME` leaking in from somewhere?

Reply:
I initially thought `JAVA_HOME` was getting incorrectly set somewhere and bypassing our `unset` as we have had issues with this in the past but that doesn't appear to be the case here. There were a few examples online of adding this `JAVA_HOME` line to the spark env file for situations where it isn't getting picked up so I took this approach.

Reply:
Sounds good. Maybe add a comment to that effect, to head off ignorant commenters like me?